pkgs/tools/security/passage/default.nix

```diff
@@ -5,6 +5,12 @@
 , substituteAll
 , age
 , getopt
+, coreutils
+, findutils
+, gnugrep
+, gnused
+, qrencode
```

Comment:
Consider `qrencode ? null`, `wl-clipboard ? null` to document the fact that they are optional.

Reply:
done